include/net/ieee802154_radio.h

```diff
@@ -157,15 +157,13 @@ struct ieee802154_radio_api {
 	u16_t (*get_subg_channel_count)(struct device *dev);
 #endif /* CONFIG_NET_L2_IEEE802154_SUB_GHZ */
 
-#ifdef CONFIG_NET_L2_OPENTHREAD
 	/** Run an energy detection scan.
-	 * Note: channel must be set prior to request this function.
-	 * duration parameter is in ms.
+	 *  Note: channel must be set prior to request this function.
```

Comment:
why the additional space?

Reply:
I just thought it looks nicer when aligned, changed w/o thinking about it too much. I can revert this change if you prefer it as it was.